### PR TITLE
Menu changes 020320

### PR DESCRIPTION
**Pull request by olhabn**
https://nanopay.atlassian.net/browse/CPF-4638

Before:
Screen Shot 2020-02-13 at 10 17 47 AM
Screen Shot 2020-02-13 at 10 18 00 AM

After:
Screen Shot 2020-02-13 at 10 14 45 AM
Screen Shot 2020-02-13 at 10 15 00 AM

### CODE REVIEW

**Comment by jlhughes** [comment]
@olhabn please create a ticket to track this work against and reference it in the PR description. 
Also, if possible, add screen shots of menuing before and after.

File: src/foam/nanos/menu/VerticalMenu.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/nanos/menu/VerticalMenu.js:18]: license header please.

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/menu/VerticalMenu.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+/**
```
Comment (kgrgreer) [inline src/foam/nanos/menu/VerticalMenu.js:1]: Use this license in the future:

/**
 * @license
 * Copyright 2017 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */

File: src/foam/nanos/menu/VerticalMenu.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          relationship: foam.nanos.menu.MenuMenuChildrenRelationship,
+          startExpanded: true,
+          query: self.menuSearch$,
+          onClickAddOn: function(data) { self.openMenu(data); },
+          formatter: function(data) { this.add(data.label); }
+        })
+      .end()
+    .end();
+
+    this.subMenu$.dot('state').sub(this.scrollToCurrentSub);
```
Comment (kgrgreer) [inline src/foam/nanos/menu/VerticalMenu.js:122]: Your indention is broken. I've fixed. Pull to get fixed version.

**Pull request merged**